docs/src/templates/pages/documentation/js/index.hbs

```diff
@@ -1,85 +1,81 @@
 ---
-title: JS Architcture
+title: JavaScript Architecture
 section-title: JS Documentation
 docs: true
 
 navgroup: documentation
 navsub: docs-js
 navactive: js-architecture
-
-wip: true
 ---
 
-{{#markdown}}
 
-<a name="philosophy"></a>
-## JavaScript Philosophy
+<a name="js-overview"></a><h2 class="sg-sectionHeading">Overiew</h2>
 
-We are currently transitioning between a legacy JavaScript structure, to a more modular dependency managed approach.
+We currently write our JavaScript using native ES2015 modules.
 
-This approach uses a tool called [Browserify](http://browserify.org/), which handles the module bundling by defining your dependencies in each of your JavaScript files.
+To help us do this we use [Browserify](http://browserify.org/), which handles module bundling through the defined dependencies in each of our JavaScript files.
 
-We also use [Babel](https://babeljs.io/) to allow us to use the latest JavaScript syntax (ES6) while still providing support for older browsers.
+We also use [Babel](https://babeljs.io/) which allows us to write code using the latest syntax and features (ES6/ES2015/ES Next) while still providing support for older browsers.  For more information on this, [see the section on Babel below](#js-babel).
 
-All of our JavaScript workflow tasks are handled by Gulp.
+All of our JavaScript workflow tasks on the main global website are currently handled by Gulp, but our JavaScript modules are used by other teams using Webpack (which we may also move to in the future).
 
-<hr class="sectionSplitter">
 
-<a name="structure"></a>
-## JavaScript Structure
+<a name="js-es2015"></a><h2 class="sg-sectionHeading">ES2015+ Ideals</h2>
 
-All of our new-style JavaScript should go into the `js/modules` folder.
+In order to help us stay up to date with the latest JavaScript features we write our modules using [the features introduced in the ES2015 release](https://github.com/lukehoban/es6features).
 
-JavaScript should be structured so that there is a clear `export` at the base of the file, so that it is clear what is being exported from the module.
+As these new features are not widely supported across the browsers that we currently support, we make use of [babelify](https://github.com/babel/babelify) & [Babel](https://github.com/babel/babel) to transform the code into ES5 and create a JavaScript bundle ready to serve in the browser.
 
-You should **always** provide a comment at the top of the module explaining what it is, how it relates to the site and examples of places it is used.
+As new JavaScript features are released beyond ES2015 (with ES2016 and ES next in the future), we must continue to evolve our compilation config using Babel to ensure that we are compiling back to a version that is acceptable for our supported browsers.
```

Comment:
This has changed since we switched to [using the env preset](https://github.com/babel/babel-preset-env) which looks at the `browserslist` property in the `package.json` file to decide which features need to be transpiled.

Reply:
I've just seen you mention this further down on the page in the Babel section. Should still update this section though I reckon.